### PR TITLE
add a new feature as  default Value

### PR DESCRIPTION
**Pull request by JerryWu1234**
#2864 
- [x] I've read and followed the [contributing guide](https://github.com/mui/mui-toolpad/blob/master/CONTRIBUTING.md#sending-a-pull-request) on how to create great pull requests.
- [x] I've updated the relevant documentation for any new or updated feature.
- [x] I've linked relevant GitHub issue with "Closes #<issue id>".
- [x] I've added a visual demonstration in the form of a screenshot or video.

### CODE REVIEW

**Comment by JerryWu1234** [comment]
@Janpot 
do you have any suggestion for this place ?
image

**Comment by JerryWu1234** [comment]
@Janpot I done this one.

File: packages/toolpad-components/src/Autocomplete.tsx [hunk 1]
```diff
@@ -91,10 +92,14 @@ function Autocomplete({
...
     }
-  }, [value]);
+    if (defaultValue) {
+      setSelectedVal(defaultValue);
+    }
+  }, [value, defaultValue]);
 
   return renderFormInput(
     <MuiAutocomplete
+      defaultValue={defaultValue}
```
Comment (Janpot) [inline packages/toolpad-components/src/Autocomplete.tsx:102]: Please, take a look at the [Select](https://github.com/mui/mui-toolpad/blob/5c6f80454d674fb91c6ed6e7864ca6e9e046fc14/packages/toolpad-components/src/Select.tsx#L123) component on how `defaultValue` is implemented. It must work the same way for Autocomplete

**Review by Janpot — APPROVED** [review]
Thanks, I aligned it a bit more to the Select component implementation.

**Pull request merged**